src/java/native/libcephfs_jni.cc

```diff
@@ -618,11 +618,10 @@ JNIEXPORT jint JNICALL Java_com_ceph_fs_CephMount_native_1ceph_1conf_1read_1file
 		return -1;
 	}
 
-	ldout(cct, 10) << "jni: conf_read_file: path " << c_path << dendl;
-
 	ret = ceph_conf_read_file(cmount, c_path);
 
-	ldout(cct, 10) << "jni: conf_read_file: exit ret " << ret << dendl;
+        ldout(cct, 10) << "jni: conf_read_file: path " << c_path <<
```

Comment:
i think, technically, it's allowed to read `conf_read_file()` multiple times. so, `log file` is not always empty when we write to `ldout()`.

Reply:
Sorry for getting back to you so late.
You're right. I checked the debug messages about java client, the log file is not empty sometimes.
When I run the hadoop test, the assert exception was thrown on the log module. 
http://tracker.ceph.com/issues/21149

I think the exception was triggered by writing the debug message when the log file is empty.

Reply:
@shangzhong i don't think the assertion of `sub < m_subsys.size())` is triggered by empty "log file". you could try out with a ceph.conf without "log file" specified.

Reply:
@tchaikov Thanks for your review. I'll do a validation later.

Reply:
@tchaikov The assertion can be reproduced. I read the related code, maybe it has nothing to do with the empty log file. 
http://tracker.ceph.com/issues/21149#note-2

Reply:
@shangzhong does the assertion go away with this change?

Reply:
@liewegas I try out with a ceph.conf without "log file" specified, the assertion can be reproduced also. Refer to http://tracker.ceph.com/issues/21149#note-2
So the assertion can't be fixed with this change. I will drop it.

Reply:
@shangzhong any update on this?

Reply:
@batrick This PR will be dropped.